action-server/actions/action_suspect_severe_symptoms_recommendations.py

```diff
@@ -0,0 +1,20 @@
+from typing import Any, Dict, List, Text
+
+from rasa_sdk import Action, Tracker
+from rasa_sdk.executor import CollectingDispatcher
+
+
+class ActionSuspectSevereSymptomsRecommendations(Action):
```

Comment:
I think this one should be generic (removing the "suspect" part of the name). Conceptually, whatever the context, if the symptoms are severe, we want to direct the user to urgency services, and there is no reason we would say it differently. And unless I am mistaken, I don't think allowing this simplification will hinder the TED policy's performance..? Anyway it's always memoization in this case.